.travis.yml

```diff
@@ -70,11 +63,125 @@ before_install:
       brew install cppunit;
       brew install c-blosc;
       brew install tbb;
-      brew install llvm@5;
+      brew install llvm@$LLVM_VERSION;
       brew install zlib;
     fi
 
+# Build matrix for linux which tests against llvm 5/6/7/8
+# for both gcc and clang. Note that the test for osx is
+# done outside of this matrix by the default settings above
```

Comment:
This comment is a little confusing because osx stuff is in this matrix, maybe say the "Note the tests for osx uses the dependencies defined above in before_install" or something similar?